2021-04-21 11:59:02,833 INFO    MainThread:11399 [wandb_setup.py:_flush():69] setting env: {}
2021-04-21 11:59:02,833 INFO    MainThread:11399 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-21 11:59:02,833 INFO    MainThread:11399 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210421_115902-2bewkld8/logs/debug.log
2021-04-21 11:59:02,834 INFO    MainThread:11399 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210421_115902-2bewkld8/logs/debug-internal.log
2021-04-21 11:59:02,834 INFO    MainThread:11399 [wandb_init.py:init():369] calling init triggers
2021-04-21 11:59:02,834 INFO    MainThread:11399 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-21 11:59:02,834 INFO    MainThread:11399 [wandb_init.py:init():418] starting backend
2021-04-21 11:59:02,834 INFO    MainThread:11399 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-21 11:59:02,846 INFO    MainThread:11399 [backend.py:ensure_launched():123] starting backend process...
2021-04-21 11:59:02,852 INFO    MainThread:11399 [backend.py:ensure_launched():128] started backend process with pid: 11424
2021-04-21 11:59:02,853 INFO    MainThread:11399 [wandb_init.py:init():423] backend started and connected
2021-04-21 11:59:02,855 INFO    MainThread:11399 [wandb_init.py:init():465] updated telemetry
2021-04-21 11:59:02,870 INFO    MainThread:11399 [wandb_init.py:init():484] communicating current version
2021-04-21 11:59:03,709 INFO    MainThread:11399 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-21 11:59:03,709 INFO    MainThread:11399 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-21 11:59:03,935 INFO    MainThread:11399 [wandb_init.py:init():522] starting run threads in backend
2021-04-21 11:59:03,956 INFO    MainThread:11399 [wandb_run.py:_console_start():1524] atexit reg
2021-04-21 11:59:03,957 INFO    MainThread:11399 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-21 11:59:03,957 INFO    MainThread:11399 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-21 11:59:03,961 INFO    MainThread:11399 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-21 11:59:03,961 INFO    MainThread:11399 [wandb_init.py:init():546] run started, returning control to user process
2021-04-21 12:00:11,726 INFO    MainThread:11399 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-21 12:00:11,728 INFO    MainThread:11399 [wandb_run.py:_restore():1466] restore
2021-04-21 12:00:13,951 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 489
  total_bytes: 489
}

2021-04-21 12:00:14,982 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 489
  total_bytes: 4591
}

2021-04-21 12:00:15,371 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4591
  total_bytes: 4591
}

2021-04-21 12:00:15,473 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4591
  total_bytes: 4591
}

2021-04-21 12:00:15,574 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4591
  total_bytes: 4591
}

2021-04-21 12:00:15,676 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4591
  total_bytes: 4591
}

2021-04-21 12:00:15,778 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4591
  total_bytes: 4591
}

2021-04-21 12:00:15,880 INFO    MainThread:11399 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4591
  total_bytes: 4591
}

2021-04-21 12:00:16,982 INFO    MainThread:11399 [wandb_run.py:_show_files():1838] logging synced files
